qa/tasks/ceph.py

```diff
@@ -1718,6 +1723,12 @@ def task(ctx, config):
             mkfs_options: [-b,size=65536,-l,logdev=/dev/sdc1]
             mount_options: [nobarrier, inode64]
 
+    To change the mdsmap's default session_timeout (60 seconds), use::
+
+        tasks:
+        - ceph:
+            session_timeout: 300
```

Comment:
```suggestion
  overrides:
    ceph:
      fs:
        session timeout: 300
```

Reply:
Let's add this to the overrides for all of the multimds suites. Add a new yaml to: qa/suites/multimds/basic/overrides and the others.

Reply:
Sure.

Reply:
BTW, do you mean should we add a `fs:` for all the other exising ones ? like the 

```
ceph:
  max_mds: 3
```
to
```
ceph:
  fs:
    max_mds: 3
```

Right ?

Reply:
Or maybe we should add `fs:` for all the fs related options, and then just do:

```
fs = Filesystem(...)
fsconf = config.get("fs", {})
fs.configure_filesytem(fsconf)
```

Will this make sense ?

Reply:
> BTW, do you mean should we add a `fs:` for all the other exising ones ? like the
> 
> ```
> ceph:
>   max_mds: 3
> ```
> 
> to
> 
> ```
> ceph:
>   fs:
>     max_mds: 3
> ```
> 
> Right ?

Yes, please!

Reply:
Usually we put these in overrides: but the difference is not significant AFAIK.

Reply:
That's right.